CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Mycha Developer Documentation
+
+#### Setup the development environment
+* install Node.JS
+* `npm install`
+
+
+#### Run tests
+* run all tests and linters: `npm test`
+* run only unit tests: `npm run unit-tests`
+* run only integration tests: `npm run feature-tests`
+
+
+#### Update dependencies
+* check whether updates are available: `npm run update-check`
+* automatically update all dependencies to the latest version: `npm run update`
```

Comment:
They get added in #61. I didn't want to take this file apart and add it in two separate halves. This is merely a copy-and-paste from other projects.